test/powershell/Language/Classes/scripting.Classes.inheritance.tests.ps1

```diff
@@ -628,3 +628,33 @@ class Derived : Base
         $sb.Invoke() | Should -Be 200
     }
 }
+
+Describe 'Base type has abstract properties' -Tags "CI" {
```

Comment:
do we have validation which ensures that an error is produced if the abstract property is _not_ defined?

Reply:
Good point. I will add one.

Reply:
A test for the error case was added.